internal/controlplane/handlers_oauth.go

```diff
@@ -218,6 +243,20 @@ func (s *Server) generateOAuthToken(ctx context.Context, provider string, code s
 		return fmt.Errorf("error exchanging code for token: %w", err)
 	}
 
+	// Older enrollments may not have a RemoteUser stored; these should age out fairly quickly.
+	tokenMatches.Add(ctx, 1, metric.WithAttributes(
```

Comment:
Should this come after we validate the user is correlated successfully? (after line 258). If we keep it here we're incrementing even if the user attempting to use the state doesn't match the user who initiated the flow.

Reply:
I wanted to know how many times when we attempt to validate a token against a user's ID.  If that's 100% of the time, then we can make the `remoteUser` field mandatory.  If it's less than that, I'll need to figure out why.

Reply:
I added a counter on the GetAuthorizationURL RPC as well, so we can correlate the number of requested tokens that store `gh_id` with the number that get claimed later.